vdp/pipeline/v1beta/connector_definition.proto

```diff
@@ -95,12 +99,21 @@ message ConnectorDefinition {
   // Connector definition custom flag, i.e., whether this is a custom
   // connector definition.
   bool custom = 11 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Connector definition icon URL.
+  // Connector definition icon URL. This is a public URL that allows clients to
+  // download and render the connector icon.
   string icon_url = 12 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
There's an ongoing discussion on Linear about how to consume the icons in the new component page (downloading the URL, with a relative path to a static asset or as a blob). I'm assuming right now we'll use the URL strategy, we can adapt later the contract to remove this field (unused now) or add a byte field.